.vscode/launch.json

```diff
@@ -11,8 +11,8 @@
             "args": [
                 "--godog.format",
                 "pretty",
-                "${workspaceFolder}/core/integration/features/liquidity-provision/0044-LIME-062.feature"
+                "${workspaceFolder}/core/integration/features/mark-price/0009-MRKP-032.feature"
```

Comment:
Can we add this file to `.gitignore`? It doesn't bother me, but it might be best for others using VSCode not to have settings tracked

Reply:
Sure, done now, though I don't think it works after a file has been added to the repo